src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
Making these internal strongly concerns me. What is the advantage of making them internal? Or similarly, the harm of keeping them public?

There currently are other encoders in this repo that call into this class for helpers. If I implement a custom encoder, why shouldn't I be able to use the methods in here as well?